RELEASING.md

```diff
@@ -0,0 +1,21 @@
+## Release checklist
+
+- [ ] Branch off of `develop` and name the branch `version-X.X.X`
+- [ ] Update version number within the repository
+  - in `setup.py`
+  - in Copyright Notice in `README`
+  - in the `LICENSE`
+- [ ] Update classifiers in `setup.py`
+- [ ] Verify that all tests pass (`python setup.py test`)
+- [ ] Commit the changes, push to GitHub, and start a pull request
+- [ ] After PR is accepted, create tagged version on GitHub
+- [ ] Add release notes to the tag on GitHub with a list of changes
+- [ ] Upload new version to PyPI
+  - `python3 -m pip install --user --upgrade setuptools wheel`
+  - `python3 setup.py sdist bdist_wheel`
+  - `python3 -m pip install --user --upgrade twine`
+  - `python3 -m twine upload dist/*`
+- [ ] Test new version installs from PyPI
+  - `cd ..`
+  - `python3 -m pip install becquerel`
+  - `python3 -c "import becquerel; print(becquerel.__version__)"`
```

Comment:
@markbandstra some more steps at the end:
- [ ] create PR from `release` branch to `develop`
- [ ] delete `release` branch

For reference this is using the methodology found in [git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)